migrations/versions/1000_nullable_clients_field.py

```diff
@@ -0,0 +1,32 @@
+"""Nullable clients field
+
+Revision ID: 1000
+Revises: 990
+Create Date: 2017-10-04 14:37:07.079122
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+
+# revision identifiers, used by Alembic.
+revision = '1000'
+down_revision = '990'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('suppliers', 'clients',
+               existing_type=postgresql.JSON(astext_type=sa.Text()),
+               nullable=True)
+    # ### end Alembic commands ###
```

Comment:
I think we've been removing the Alembic-autogenerated comments on upgrade and downgrade - definitely not a blocker though!

Reply:
Oh I was planning to blame Alembic if things broke! :trollface: